### DISCUSSION TITLE
How did you make the webgl demo?

### DISCUSSION THREAD

**Discussion by YoonInha** [atteneder/glTFast#628] [Q&A]
The example source uses threads, but I know they are not supported on the web. Looking at the function types, they all return tasks and use async and await. As a result of several attempts, the code that works in the local editor environment does not work when building the web. But you made a demo on the web, can I get the source code for this or the reference document for the function? I am in a situation where I receive a model file from a remote file server by url and process it in webgl. thank you.

**Comment by YoonInha** [comment]
I was doing it with version 5 of GLTFast, but I found out that the webgl version uses version 4.8. Also, I belatedly confirmed that there was a link to the webgl project on the demo page. thank you~

**Reply by JimmyAkl** [reply]
Hello, is it possible to share how it's done on webgl ? thank you!

**Comment by atteneder** [comment]
glTFast is known to work on WebGL, despite usinc async/await. The reason it works is Unity's custom synchronisation context.

Unfortunately the web demo is kinda outdated at the moment. I hope I'll find time to update it at some point.
